test/jobs/job-status-pagination.ts

```diff
@@ -0,0 +1,209 @@
+import { expect } from 'chai';
+import { describe, it, before } from 'mocha';
+import { Job } from 'models/job';
+import JobLink from 'models/job-link';
+import hookServersStartStop from '../helpers/servers';
+import { hookJobStatus, buildJob, areJobLinksEqual, itIncludesPagingRelations } from '../helpers/jobs';
+import db from '../../app/util/db';
+import env from '../../app/util/env';
+
+describe('Individual job status route - pagination', function () {
+  hookServersStartStop({ skipEarthdataLogin: false });
+
+  const links: JobLink[] = [] as JobLink[];
+  const aJob = buildJob({ username: 'joe', links });
+  let defaultPageSize;
+
+  before(async function () {
+    // use a DEFAULT_PAGE size of 10 for these tests
+    ({ defaultPageSize } = env);
+    env.defaultPageSize = 10;
+
+    // Generate some links for the job - need to save the job after each link is added to
+    // circumvent ordering problems with sqlite
+    const jTrx = await db.transaction();
+    await aJob.save(jTrx);
+    jTrx.commit();
+    for (let i = 1; i < 51; i++) {
```

Comment:
I'm not sure how fast these tests run, but consider changing the default page size to say 5 and then 20 links if the test is slightly slow.

Reply:
I didn't notice any slowdown.